app.js

```diff
@@ -1,47 +1,147 @@
 console.log('Starting a project!')
 
 const secondsElement = document.getElementById('seconds');
-const minutesElement = document.getElementById('minutes');
-const hoursElement = document.getElementById('hours');
+const milisecondsElement = document.getElementById('miliseconds');
 const startAndResetButtonElement = document.getElementById('start');
 const boardElement = document.getElementById('board');
+let scoreBoardElement = document.getElementById('score-board');
+let lifeBoardElement = document.getElementById('lives')
+let score = 0;
 let intervalId;
-let totalseconds = 0;
+let totalmiliseconds = 300;
 let isTimerRunning = false;
+let life = 3;
+let width = 50;
+let height = 50;
 
 function pad(val){
   const value = val + "";
-  if(value.length < 2){
+  if (value.length < 2) {
     return "0" + value;
   } else {
     return value;
   }
 }
 
-function setTime(){
-  ++totalseconds;
-  secondsElement.innerHTML = pad(totalseconds) - (minutesElement.innerHTML*60) - (hoursElement.innerHTML*3600);
-  minutesElement.innerHTML = pad(parseInt(totalseconds/60)) - (hoursElement.innerHTML*60);
-  hoursElement.innerHTML = pad(parseInt(totalseconds/3600));      
+function createTarget() {
+  const boardParams = boardElement.getBoundingClientRect();
+  let x=Math.random()*(boardParams.width - 50);
```

Comment:
static typed `50` number is incorrect as the boxes will be reduced in size when the game is running